shared/hybrid/AppDelegate+SalesforceHybridSDK.m

```diff
@@ -59,23 +59,23 @@ - (AppDelegate *)sfsdk_swizzled_init
     // Need to use SalesforceSDKManagerWithSmartStore when using smartstore
     [SalesforceSDKManager setInstanceClass:[SalesforceSDKManagerWithSmartStore class]];
     [SalesforceSDKManager sharedManager].appConfig = appConfig;
-    __weak typeof(self) weakSelf = self;
+    __weak __typeof(self) weakSelf = self;
     [SalesforceSDKManager sharedManager].postLaunchAction = ^(SFSDKLaunchAction launchActionList) {
-        __strong typeof(weakSelf) strongSelf = weakSelf;
-        [strongSelf log:SFLogLevelInfo format:@"Post-launch: launch actions taken: %@", [SalesforceSDKManager launchActionsStringRepresentation:launchActionList]];
-        [strongSelf setupRootViewController];
+      __strong __typeof(weakSelf) strongSelf = weakSelf;
```

Comment:
The spacing went wonky from the standard spacing that Xcode uses.  Can you Editor -> Structure -> Re-Indent these?